### PR TITLE
[DO NOT MERGE] Release 5.3.0-customercenter.alpha.4

### PR DESCRIPTION
**Pull request by vegaro**
Preparing release base off bb71e05f4fc9969e7f6256df390f4e487b8a0b03

* Fix `FeedbackSurveyView` not opening (#4208) via Cesar de la Vega (@vegaro)
* Migrate to List style (#4190) via Cody Kerns (@codykerns)
* Hide Contact Support button if URL can't be created (#4192) via Cesar de la Vega (@vegaro)
* Improving customer center buttons (#4165) via Cody Kerns (@codykerns)
* Clean up colors in WrongPlatformView and NoSubscriptionsView (#4204) via Cesar de la Vega (@vegaro)
* CustomerCenterViewModel checks whether the app is the latest version (#4169) via JayShortway (@JayShortway)

### CODE REVIEW

**Review by JayShortway — APPROVED** [review]

File: RevenueCatUI/CustomerCenter/Views/CompatibilityTopBarTrailing.swift [hunk 1]
```diff
@@ -12,6 +12,7 @@
 //  Created by Josh Holtz on 8/16/24.
 
 #if CUSTOMER_CENTER_ENABLED
+#if os(iOS)
```
Comment (tonidero) [inline RevenueCatUI/CustomerCenter/Views/CompatibilityTopBarTrailing.swift:15]: Should we port this to `main`?

File: Tests/APITesters/AllAPITests/SwiftAPITester/CustomerCenterConfigDataAPI.swift [hunk 2]
```diff
@@ -16,12 +16,14 @@ func checkCustomerCenterConfigData(_ data: CustomerCenterConfigData) {
     let localization: CustomerCenterConfigData.Localization = data.localization
     let support: CustomerCenterConfigData.Support = data.support
     let lastPublishedAppVersion = data.lastPublishedAppVersion
+    let productId = data.productId
```
Comment (tonidero) [inline Tests/APITesters/AllAPITests/SwiftAPITester/CustomerCenterConfigDataAPI.swift:19]: Same about these APIs

**Review by tonidero — APPROVED** [review]

File: RevenueCatUI/CustomerCenter/Views/CompatibilityTopBarTrailing.swift (hunk 1 above)
Comment (vegaro) [reply RevenueCatUI/CustomerCenter/Views/CompatibilityTopBarTrailing.swift:15]: yes, they were in #4226 which I just merged to main

File: Tests/APITesters/AllAPITests/SwiftAPITester/CustomerCenterConfigDataAPI.swift (hunk 2 above)
Comment (vegaro) [reply Tests/APITesters/AllAPITests/SwiftAPITester/CustomerCenterConfigDataAPI.swift:19]: yes, they were in #4226 which I just merged to main

**Pull request closed without merging**